schema.py

```diff
@@ -0,0 +1,251 @@
+from typing import Dict, Optional, List, Literal, Any, Union
+
+from pydantic import BaseModel, AnyHttpUrl, EmailStr, Field
+
+ASYNCAPI_VERSION = '2.0.0'
```

Comment:
Do we want to start with this version? We probably want to be compliant with the latest one i.e. `2.3.0`.

Reply:
Yes! We want 2.3.0.
I saw 2.0.0 and didn't change anything.
I will change that.